lib/choice.fz

```diff
@@ -79,12 +79,12 @@
 # Now we define the choice type using the wrapped i32 types, which are
 # distinct:
 #
-#   temp centigrade | fahrenheit := ...
+#   hasFever(temp centigrade | fahrenheit) bool is ...
 #
-# When assigning values to temp, we need to wrap the values accordingly:
+# When giving values to this feature, we need to wrap them accordingly:
```

Comment:
@maxteufel What do you think about this:
~~When giving values to this feature~~
When passing arguments to this feature